### PR TITLE
docs: add routes and files

### PR DESCRIPTION
**Pull request by D0m1nu2**
## Summary
Add A380 active routes overview for FBW website. Currently I've only made few of them to see if it's ok like this, once I got confirmation, I will finish the rest of them.
I've used FlightRadar24 for route length and average flight time, then Planespotter for fleet info.

### Location
docs/fbw38nx/xxx

Discord username (if different from GitHub): D0m1nuZ @D0m1nu2

### CODE REVIEW

**Review by Valastiri — CHANGES_REQUESTED** [review]
A few things but to start thanks for continuing work on this.

## Minor Notes
- We have a WIP directory that includes some A380 flight ops information. If you can move the entirety of your PR's contents to that folder and keep your structure for easy moving later that would be great as we aren't ready to publish this info just yet but I would gladly have this on the repo. 
	- For the above when we update our installation in the coming days you can most likely leave your documentation where it is and we can exclude it from the active build until the A380 is ready OR we have other ideas of when exactly to present this information.
- If you'd like to refactor / update the existing information alongside yours that would be helpful otherwise we can leave that for a later time.

## Critical
- Your images are too large and do not conform to our standards. 
- See [Image Assets Process](https://docs.flybywiresim.com/dev-corner/development-projects/documentation/?h=documentation+project#image-assets-process) on our guide.

File: docs/fbw-a380x/routes/operating/QTR.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+---
+
+## ^^Routes^^
+Qatar Airways is currently operating their A380s on 6 routes to 4 destinations.
+
+![QTR Routes](../../../fbw-a38nx/assets/routes/qtrroutes.jpg "QTR Routes")
+
+### **Currently operating**
+| Departure ICAO/IATA | Arrival ICAO/IATA | Flight Number | Lenght | Distance |
```
Comment (Valastiri) [inline docs/fbw-a380x/routes/operating/QTR.md:30]:
```suggestion
| Departure ICAO/IATA | Arrival ICAO/IATA | Flight Number | Length | Distance |

```

Comment (Valastiri) [reply docs/fbw-a380x/routes/operating/QTR.md:30]: Although I would suggest changing "Length" -> "Duration" // as that would make more sense in aviation to me.

Additionally, length is misspelled in each document.

**Review by Sleinmaster — CHANGES_REQUESTED** [review]
You'll have to correct the units. I can make a review with suggestions in the coming days, but here is essentially what has to be changed.

- Between the numbers and the respective units you should insert a non-breakable space (alt+0160 on windows).
- the unit for nautical miles is upper case, so NM.

File: docs/fbw-a380x/routes/operating/AAR.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Asiana Airlines (AAR/OZ)
+
+---
+
+## ^^Fleet^^
+Asiana Airways has 6 A380s in it's fleet first delivered in December 2013 and last on August 2016
```
Comment (Sleinmaster) [inline docs/fbw-a380x/routes/operating/AAR.md:6]:
```suggestion
Asiana Airways has 6 A380 in its fleet first delivered in December 2013 and last on August 2016

```

File: docs/fbw-a380x/routes/operating/AAR.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+### HL7626
+### HL7634
+### HL7635
+### HL7640
+### HL7641
+
+---
+
+## ^^Routes^^
+Asiana is currently operating their A380s on 3 routes around the world.
```
Comment (Sleinmaster) [inline docs/fbw-a380x/routes/operating/AAR.md:20]:
```suggestion
Asiana is currently operating their A380 on 3 routes around the world.

```

File: docs/fbw-a380x/routes/operating/AAR.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+### **Currently operating**
+| Departure ICAO/IATA | Arrival ICAO/IATA | Flight Number | Lenght | Distance |
+|:--------|:--------|:------|:------|:-----|
+| RKSI/ICN | VTBS/BKK | OZ741 | 5h15m | 1980nm |
+| VTBS/BKK | RKSI/ICN | OZ742 | 4h40m | 1980nm |
+| RKSI/ICN | YSSY/SYD | OZ601 | 10h00m | 4550nm |
+| YSSY/SYD | RKSI/ICN | OZ602 | 10h10m | 4550nm |
+| RKSI/ICN | KLAX/LAX | OZ202 | 10h10m | 5210nm |
+| KLAX/LAX | RKSI/ICN | OZ201 | 12h55m | 5210nm |
```
Comment (Sleinmaster) [inline docs/fbw-a380x/routes/operating/AAR.md:32]:
```suggestion
| Departure ICAO/IATA | Arrival ICAO/IATA | Flight Number | Length | Distance |
|:--------|:--------|:------|:------|:-----|
| RKSI/ICN | VTBS/BKK | OZ741 | 5 h 15 m   | 1980 NM |
| VTBS/BKK | RKSI/ICN | OZ742 | 4 h 40 m   | 1980 NM |
| RKSI/ICN | YSSY/SYD | OZ601 | 10 h 00 m | 4550 NM |
| YSSY/SYD | RKSI/ICN | OZ602 | 10 h 10 m | 4550 NM |
| RKSI/ICN | KLAX/LAX | OZ202 | 10 h 10 m | 5210 NM |
| KLAX/LAX | RKSI/ICN | OZ201 | 12 h 55 m | 5210 NM |

```

**Review by Sleinmaster — CHANGES_REQUESTED** [review]
This as an example, please change the others accordingly and don't forget to format the tables.

**Comment by Valastiri** [comment]
@D0m1nu2 is there any update on this PR?

**Comment by D0m1nu2** [comment]
> @D0m1nu2 is there any update on this PR?

I'm sorry for my recent inactivity, I did not have much time to look into this due to big amount of work. I will try to find some more time in the upcoming weeks so I fix and add all the mentioned things.

**Comment by Valastiri** [comment]
> > @D0m1nu2 is there any update on this PR?
> 
> I'm sorry for my recent inactivity, I did not have much time to look into this due to big amount of work. I will try to find some more time in the upcoming weeks so I fix and add all the mentioned things.

No worries -- just wanted to check in! I have updated my review regarding the WIP directory and whenever you are ready for review I might simply add a few commits to ensure your documentation is appropriately setup if it's out of scope for you.

Either way thanks again for doing this good stuff.

**Comment by Valastiri** [comment]
Outdated

**Pull request closed without merging**